self-serve-storage/python/Storage Examples.ipynb

```diff
@@ -39,17 +45,27 @@
    "outputs": [],
    "source": [
     "import daaas_storage\n",
-    "import os\n",
-    "#minio_client = daaas_storage.get_minimal_client()\n",
-    "minio_client = daaas_storage.get_premium_client()\n",
-    "#minio_client = daaas_storage.get_pachyderm_client()"
+    "\n",
+    "storage = daaas_storage.get_minimal_client()\n",
+    "# minio_client = daaas_storage.get_premium_client()"
```

Comment:
This is my bad, but I think we should use `minioClient` so that it's consistent with the examples [here](https://docs.min.io/docs/python-client-api-reference)

Reply:
I think there are two types of users:

* Ones who aren't comfortable peering under the hood: "Minio" is not meaningful to them, nor is the associated client's API
* Ones who are comfortable: they understand that the `daaas_storage.py` is returning a Minio instance that is being stored in a variable that can take any name

That was the reasoning. Revert if you'd like. 👍